packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.stories.ts

```diff
@@ -0,0 +1,348 @@
+        lastName: 'Bouvier',
+        favoriteColor: 'Teal'
+    },
+    {
+        id: 'Selma Bouvier',
+        firstName: 'Selma',
+        lastName: 'Bouvier',
+        favoriteColor: 'Lavender'
+    },
+    {
+        id: 'Nelson Muntz',
+        firstName: 'Nelson',
+        lastName: 'Muntz',
+        favoriteColor: 'Magenta'
+    },
+    {
+        id: 'Jimbo Jones',
+        firstName: 'Jimbo',
+        lastName: 'Jones',
+        favoriteColor: 'Cyan'
+    },
+    {
+        id: 'Kearney Zzyzwicz',
+        firstName: 'Kearney',
+        lastName: 'Zzyzwicz',
+        favoriteColor: 'Azure'
+    },
+    {
+        id: 'Dolph Starbeam',
+        firstName: 'Dolph',
+        lastName: 'Starbeam',
+        favoriteColor: 'Aquamarine'
+    },
+    {
+        id: 'Wendell Borton',
+        firstName: 'Wendell',
+        lastName: 'Borton',
+        favoriteColor: 'Turquoise'
+    },
+    {
+        id: 'Martin Prince',
+        firstName: 'Martin',
+        lastName: 'Prince',
+        favoriteColor: 'Sienna'
+    },
+    {
+        id: 'Sherri Mackleberry',
+        firstName: 'Sherri',
+        lastName: 'Mackleberry',
+        favoriteColor: 'Periwinkle'
+    },
+    {
+        id: 'Terri Mackleberry',
+        firstName: 'Terri',
+        lastName: 'Mackleberry',
+        favoriteColor: 'Coral'
+    },
+    {
+        id: 'Rod Flanders',
+        firstName: 'Rod',
+        lastName: 'Flanders',
+        favoriteColor: 'Cyan'
+    },
+    {
+        id: 'Todd Flanders',
+        firstName: 'Todd',
+        lastName: 'Flanders',
+        favoriteColor: 'Magenta'
+    },
+    {
+        id: 'Agnes Skinner',
+        firstName: 'Agnes',
+        lastName: 'Skinner',
+        favoriteColor: 'Azure'
+    }
+] as const;
+
+const metadata: Meta<SharedTableArgs> = {
+    title: 'Components/Table Column: Menu Button',
+    decorators: [withActions<HtmlRenderer>],
+    parameters: {
+        actions: {
+            handles: [
+                ...sharedTableActions,
+                'menu-button-column-toggle',
+                'menu-button-column-beforetoggle'
+            ]
+        }
+    },
+    // prettier-ignore
+    argTypes: {
+        ...sharedTableArgTypes,
+        selectionMode: {
+            table: {
+                disable: true
+            }
+        },
+    },
+    args: {
+        ...sharedTableArgs(simpleData)
+    }
+};
+
+export default metadata;
+
+interface MenuButtonColumnTableArgs extends SharedTableArgs {
+    fieldName: string;
+    menuSlot: string;
+    updateMenuItems: (
+        storyArgs: MenuButtonColumnTableArgs,
+        e: CustomEvent<MenuButtonColumnToggleEventDetail>
+    ) => void;
+    menuRef: Menu;
+    toggleEvent: never;
+    beforeToggleEvent: never;
+    currentData: TableRecord[];
+}
+
+export const menuButtonColumn: StoryObj<MenuButtonColumnTableArgs> = {
+    parameters: {},
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<MenuButtonColumnTableArgs>`
+        <${tableTag}
```

Comment:
In the docs view the menu button isn't able to pop over the content. In combobox / select / menubutton we fix that using this helper:
https://github.com/ni/nimble/blob/2d017ba9169c8d83fb72bf98d6a5f657bd6f6e8d/packages/storybook/src/nimble/combobox/combobox.stories.ts#L106